src/text.c

```diff
@@ -380,30 +437,53 @@ bool32 AddTextPrinter(struct TextPrinterTemplate *printerTemplate, u8 speed, voi
 
 void RunTextPrinters(void)
 {
-    int i;
+    bool32 isInstantText = IsPlayerTextSpeedInstant(); // Force correct result. This is dumb, Revo knows.
```

Comment:
```suggestion
    bool32 isInstantText = GetPlayerTextSpeed() == OPTIONS_TEXT_SPEED_INSTANT
```

Isn't this the same thing?

Also `This is dumb, Revo knows` is not needed

Reply:
no, since it can be overridden with the flag or the setting

Reply:
```C
u32 GetPlayerTextSpeed(void)
{
    if (gTextFlags.forceMidTextSpeed)
        return OPTIONS_TEXT_SPEED_MID;

    if (gSaveBlock2Ptr->optionsTextSpeed > OPTIONS_TEXT_SPEED_INSTANT)
        gSaveBlock2Ptr->optionsTextSpeed = OPTIONS_TEXT_SPEED_FAST;

    if (FlagGet(FLAG_TEXT_SPEED_INSTANT) || TEXT_SPEED_INSTANT)
        return OPTIONS_TEXT_SPEED_INSTANT;

    return gSaveBlock2Ptr->optionsTextSpeed;
}
```
Isn't it overridden here as well?